app/src/main/java/com/example/contactsapp/MyContactRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.contactsapp
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.contactsapp.databinding.FragmentContactItemBinding
+
+class MyContactRecyclerViewAdapter(
+    private val mOnContactClickListener: OnContactClickListener,
+    private val values: List<ContactData>
+) : RecyclerView.Adapter<MyContactRecyclerViewAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+
+        return ViewHolder(
+            FragmentContactItemBinding.inflate(
+                LayoutInflater.from(parent.context),
+                parent,
+                false
+            ),
+            mOnContactClickListener
```

Comment:
`mOnContactClickListener` uses outdated naming style, just write the `onContactClickListener`